test/implementations/node.js

```diff
@@ -1,31 +1,9 @@
-// const mongoUnit = require('mongo-unit')
-// const spawn = require('child_process').spawn
-// const exec = require('child_process').exec
-
 import level from 'level'
-import memdown from 'memdown'
 
 export default [
   {
     key: 'level',
     type: 'node',
```

Comment:
Not sure of the use of 'type' here. I looked through the code for any usage of implementation.type but couldn't find any. Also, the new Level v8 can differentiate between ClassicLevel (I.e. running in node) and BrowserLevel (running in browser) and load the database in the appropriate manner.

Reply:
You're right! I removed the implementations and cleaned it up to reflect the new leveldb usage.